integration-tests/spring-di/src/main/java/io/quarkus/it/spring/GreeterBean.java

```diff
@@ -18,11 +18,11 @@
     @Qualifier("cap")
     StringFunction capitalizerStringFunction;
 
-    @Value("${greeting.suffix:!}")
-    String suffix;
```

Comment:
Do we still have a test for that elsewhere?

Reply:
It sure does: https://github.com/quarkusio/quarkus/blob/master/integration-tests/spring-di/src/main/java/io/quarkus/it/spring/MessageProducer.java#L8